src/Tizen.NUI/src/public/BaseComponents/ModelView.cs

```diff
@@ -15,202 +15,243 @@
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView() : this(Interop.ModelView.ModelViewNew(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
         }
-        public Model3dView(string objUrl, string mtlUrl, string imagesUrl) : this(Interop.Model3DView.Model3dViewNew(objUrl, mtlUrl, imagesUrl), true)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView(string objUrl, string mtlUrl, string imagesUrl) : this(Interop.ModelView.ModelViewNew(objUrl, mtlUrl, imagesUrl), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
         }
-        public Model3dView(Model3dView model3dView) : this(Interop.Model3DView.NewModel3dView(Model3dView.getCPtr(model3dView)), true)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView(ModelView ModelView) : this(Interop.ModelView.NewModelView(ModelView.getCPtr(ModelView)), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
-        public Model3dView Assign(Model3dView model3dView)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView Assign(ModelView ModelView)
         {
-            Model3dView ret = new Model3dView(Interop.Model3DView.Model3dViewAssign(SwigCPtr, Model3dView.getCPtr(model3dView)), false);
+            ModelView ret = new ModelView(Interop.ModelView.ModelViewAssign(SwigCPtr, ModelView.getCPtr(ModelView)), false);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
-        public static Model3dView DownCast(BaseHandle handle)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ModelView DownCast(BaseHandle handle)
         {
-            Model3dView ret = Registry.GetManagedBaseHandleFromNativePtr(handle) as Model3dView;
+            ModelView ret = Registry.GetManagedBaseHandleFromNativePtr(handle) as ModelView;
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
could you check the warning in line95?
need to use PascalCase.
=>
**_src/Tizen.NUI/src/public/BaseComponents/ModelView.cs#L95
Remove the underscores from member name_**